_posts/2017-09-01-the-scourge-of-silos.markdown

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title:  "The Scourge of Silos"
+date:   2017-09-01 04:48:58 +0000
+categories: bdd tdd devops pairing mobbing
+author: Hamish Tedeschi
+image: img/silos1.png
+---
+This is one of those articles that may stir up a bit of angst with people, because essentially it is an opinion piece. Not driven by data, merely driven by what I think works best. It was provoked by a brief conversation on social media and got me thinking about why I think the way I do. The conversation was based on the following post by [Josh Partogi](https://www.linkedin.com/in/jpartogi/), which was on the money:
```

Comment:
Ok so this first paragraph is good conversational style to get started.  However, looking at the SEO crap from WP, it mentioned that the first sentence should sum up the article to bump up the ratings.

Reply:
Also, the link here is just to his profile, not the post - can you link to the post itself?  

Or is the quote the whole post? - confusing for me